MobileDevProject/ViewController.swift

```diff
@@ -111,7 +102,7 @@ class ViewController: UIViewController, CLLocationManagerDelegate {
             var count: Int = 0
             for beacon in beaconInfo {
                 var buttonColour: UIColor
-                let colourAmount = (255 - (CGFloat(beacons[count].accuracy) * 40))
+                let colourAmount = (255 - (CGFloat(beacons[0].accuracy) * 40))
```

Comment:
This should still be beacons[count] unless we're removing having all buttons change colour